istioctl/pkg/tag/generate.go

```diff
@@ -265,10 +265,14 @@ func mergeMaps(base, override map[string]string) map[string]string {
 	if override == nil {
 		return base
 	}
+	merged := make(map[string]string)
```

Comment:
The `mergeMaps` function was originally intended for merging labels, and it's not very necessary to allocate new space in this case. If you think its misunderstandable, I suggest revising it to utilize this `mergeMaps`:
https://github.com/istio/istio/blob/8652668835700c4e278dac4437a9e736602bc559/pkg/kube/inject/template.go#L386-L398
This would also standardize its use throughout the package

Reply:
I found that we can reuse the maps package, The function of Copy in this package is the same as this function, and will not affect the original logic.

Reply:
Looks good